CHANGELOG.md

```diff
@@ -12,6 +12,18 @@
 
 ---
 
+[//]: # (START/v1.9.0)
+# v1.9.0
+
+## Features
+  * Operator can be used with 1Password Service Accounts now. {#160}
+
+## Fixes
+ * Update Operator to use SDK v1.34.1. {#185}
```

Comment:
`[non-blocking suggestion]`

This happened so long ago that I'm considering doing another upgrade. The latest is `1.40.0` ([link](https://sdk.operatorframework.io/docs/upgrading-sdk-version/)) and while this is an additive element, we should keep our operator up-to-date as a good practice.

Reply:
Good point. Created [an issue](https://github.com/1Password/onepassword-operator/issues/207) to track it.